clkhash/rest_client.py

```diff
@@ -4,9 +4,13 @@
 import clkhash
 from clkhash.backports import TimeoutError
 import logging
+from retrying import retry
 
 logger = logging.getLogger(__name__)
 
+WAITING_TIME_WHEN_RATE_LIMITED_MS = 2000  # 1 seconds
```

Comment:
This looks like 2 seconds
```suggestion
WAITING_TIME_WHEN_RATE_LIMITED_MS = 2000
```